CHANGELOG.rst

```diff
@@ -0,0 +1,118 @@
+  every data-type that the standard allows for curves is now supported by
+  dlisio.
+* ``Frame``- and ``Channel``-objects now have an index-property. ``index`` returns the
+  ``Channel``-object that serves as the index-channel for the given Frame/Channel.
+
+0.1.13_ - 2019.10.3
+-------------------
+* The sphinx documentation on readthedocs_ has a few new section: About the
+  project, an introduction to some dlis-concepts and a quick guide to help new
+  users to get started with dlisio.
+* API documentation has seen some improvements as well. The ``dlis``-class
+  documentation is revamped to better help users to work with logical files and
+  accessing objects. ``Frame`` and ``Channel`` are more thoroughly documented, and
+  more examples on how to work with curve data are provided.
+* Direct access to a specific objects has been made more convenient with
+  ``dlis.object()``.
+* ``dlis.match()`` is no longer case sensitive.
+* ``dlis.fileheader`` now returns the ``Fileheader``-object directly, not wrapped as
+  dict_values.
+* ``dlis.objects`` has been removed
+* CircleCI is added to the ci-pipeline for building and testing on linux
+* Python test suite has seen some refactoring
+* It is now possible to build the python module with ``setup.py``, provided the
+  core library is already installed on the system.
+
+0.1.12_ - 2019.08.15
+--------------------
+* Output a readable summary of any metadata-object, logical file or batch-object
+  with ``.describe()``.
+* Access to curves directly through ``Frame``- and ``Channel``-objects.
+* dlisio has learned to read the following metadata-objects: ``Process``, ``Path``,
+  ``Splice``, ``Well reference point``, ``Group``, ``Message``, ``Comment``.
+* ``dlis.match()`` lets you search for objects with a regular expression.
+* dlisio now reads even more files. Restrictions such as number-of-objects in an
+  object_set and missing representation codes in templates have been lifted.
+* The parsing routine has seen some improvements. This includes giving the user
+  more freedom to customize object-parsing.
+* Multidimensional metadata attributes are handled correctly.
+* ``BasicObject.update_stash`` has been removed.
+* ``dlis.getobjects()`` has been removed.
+* ``dlis.object_set`` has been renamed to ``dlis.indexedobjects``.
+* ``Computation.source`` is now a scalar, not vector.
+* ``BasicObject``'s ``type`` and ``attic`` is now attributes, not properties.
+* Objects are allowed to have empty ids (name/mnemonic).
+* The API documentation has seen some minor updates.
+* dlisio uses endianness.h rather than its own implementation.
+* Some of the binary test files have been simplified.
+* core functionality such as ``findfdata``, ``findsul``, ``findvrl``, ``findoffsets`` and
+  ``stream.at`` are more thoroughly tested.
+* Parts of the Python test suite have been refactored.
+* Fixed a bug where long obnames where allocated insufficient memory.
```

Comment:
long obnames (were) allocated
For all the typos in the commit message I had to find some I can comment on :laughing: